pkg/api/api_implementation.go

```diff
@@ -79,8 +88,19 @@ func NewAPI(opts ...NewAPIOption) (API, error) {
 // Get the identified object from the engine.
 func (a defaultAPI) Get(ctx context.Context, o types.IdentifiedObject, opts ...types.GetOption) error {
 	options := types.GetOptions{}
+	var err error
+	for _, requestOpt := range a.requestOptions {
+		if getOpt, ok := requestOpt.(types.GetOption); ok {
+			err = errors.Join(err, getOpt.ApplyToGet(&options))
```

Comment:
I would've just returned on the first option returning an error, but I guess collecting all of them could be useful in some situations - "ok, would have done differently tho" ^^

Reply:
Did that to make codeclimate less angry (too many returns) :grimacing: